api/verify.go

```diff
@@ -219,7 +219,7 @@ func (a *API) verifyPost(w http.ResponseWriter, r *http.Request) error {
 			return terr
 		}
 
-		token, terr = a.issueRefreshToken(ctx, tx, user, models.SMSOrGeneratedLink, grantParams)
+		token, terr = a.issueRefreshToken(ctx, tx, user, models.SMSOrGeneratedLink.String(), grantParams)
```

Comment:
this could be an email OTP too

Reply:
Will change this to SMSOrEmailOTP since GeneratedLink produces an EmailOTP